web/docs/auth/overview.md

```diff
@@ -504,8 +504,382 @@ try {
 </TabItem>
 </Tabs>
 
+## Customizing the Signup Process
+
+Sometimes you want to include **extra fields** in your signup process, like first name and last name.
+
+In Wasp, in this case:
+
+- You need to define the new backend fields
+- You need to customize the `SignupForm`
+
+Other times, you might need to add some **extra UI** elements to the form, like a checkbox for terms of service. In this case, customizing only the UI components is enough.
+
+Let's see how to do both.
+
+### 1. Defining Extra Backend Fields
+
+If we want to **save** some extra fields in our signup process, we need to tell our backend they exist.
+
+We do that by defining an object where the keys represent the field name, and the values are functions that receive the data sent from the client and return the value of the field.
+
+First, we add the `auth.signup.additionalFields` field in our `main.wasp` file:
```

Comment:
```suggestion
First, we add the `auth.signup.additionalFields` field in our `main.wasp` file, via which we provide the path to the JS/TS file where we are going to define these extra fields.
```

Reply:
You don't think it is good? The reason I put this is because I found it unclear as to which value that field should have. You set it to `"@server/auth/signup.js"`, but why, what is the intention? Does it have to that file name, can it be something else?
If you feel it is clear, ok, leave it as it is.

Reply:
Based on the "Show, don't tell" principle, this sentence feels redundant. It's clear from the context that we are putting a JS/TS file there and it's clear that in that file we are defining the fields.

Reply:
To me that wasn't clear from the context, not until I read further along. But ok, I will leave it to you.